odoo/addons/base/models/ir_filters.py

```diff
@@ -41,7 +42,10 @@ def copy(self, default=None):
     @api.multi
     def _get_eval_domain(self):
         self.ensure_one()
-        return ast.literal_eval(self.domain)
+        return safe_eval(self.domain, {
+            'datetime': datetime,
+            'context_today': datetime.datetime.now,
```

Comment:
context_today is on Date and will return a date instead of the datetime.
And prefer to stay coherent with all others occurences in crm: https://github.com/odoo/odoo/blob/7fecbc91eb74f277b2ea5885217dc48811f527ee/addons/website_crm_score/models/website_crm_score.py#L12

But yes, we could image to change all at the same time in master.